src/test/java/core/mvc/JsonViewTest.java

```diff
@@ -24,13 +24,13 @@ public class JsonViewTest {
     void setUp() {
         request = new MockHttpServletRequest();
         response = new MockHttpServletResponse();
-        view = new JsonView();
+        view = new JsonView("/");
     }
 
     @Test
     void render_no_element() throws Exception {
         view.render(new HashMap<>(), request, response);
-        assertThat(response.getContentType()).isEqualTo(MediaType.APPLICATION_JSON_VALUE);
+        assertThat(response.getContentType()).isEqualTo(MediaType.APPLICATION_JSON_UTF8_VALUE);
```

Comment:
```suggestion
        assertThat(response.getContentType()).isEqualTo(MediaType.APPLICATION_JSON.toString());
```
contentType 이 String 을 반환하고 있기 때문에 이렇게 해야 테스트를 통과하는 것 같아요 :)

Reply:
아아 짚어주셨군요 옙옙 수정해서 푸시하겠습니다!!

Reply:
같은 타입인 MediaType.APPLICATION_JSON_VALUE 으로 처리했습니다!!